bar-trender-front-end/src/views/ListView.js

```diff
@@ -10,13 +10,13 @@ import "./css/FilterResults.css"
 
 // core components
 
-import ExamplesNavbar from "../components/Navbars/ExamplesNavbar.js";
 import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
 import List from "../components/List";
 import withListLoading from '../components/withListLoading';
 import DeviceIdentifier from "react-device-identifier";
+import MainNavbar from 'components/Navbars/MainNavbar';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7163535)
```suggestion
import MainNavbar from "components/Navbars/MainNavbar";
```